astropy/modeling/tests/test_blackbody.py

```diff
@@ -130,16 +130,12 @@ def test_blackbody_exceptions_and_warnings():
     assert exc.value.args[0] == 'Temperature should be positive: -100.0 K'
 
     # Zero wavelength given for conversion to Hz
-    with catch_warnings(AstropyUserWarning) as w:
+    with pytest.warns(AstropyUserWarning, match='invalid'):
         blackbody_nu(0 * u.AA, 5000)
-    assert len(w) == 1
```

Comment:
There are actually more than 1 warning because of the deprecation warning. Since `blackbody.py` itself is deprecated, I am not worried about how many warnings exactly here.